homeworks/src/homework8/FileMaxSizeReachedException.java

```diff
@@ -0,0 +1,11 @@
+package homework8;
+
+import java.io.File;
+
+public class FileMaxSizeReachedException extends Throwable {
```

Comment:
it's better to extend Exception or RuntimeException. they are provide constructor methods that accept a Throwable which describes the cause of the exception